llvm/test/CodeGen/SPIRV/hlsl-intrinsics/splitdouble.ll

```diff
@@ -0,0 +1,50 @@
+; RUN: llc -verify-machineinstrs -O0 -mtriple=spirv-unknown-unknown %s -o - | FileCheck %s
+; RUN: %if spirv-tools %{ llc -O0 -mtriple=spirv-unknown-unknown %s -o - -filetype=obj | spirv-val %}
+
+; Make sure lowering is correctly generating spirv code.
```

Comment:
This just tests SPIR-V lowering of the code you expect to generate from the HLSL.  It doesn't actually test that the `CGBuiltin.cpp` code generates this IR for the SPIR-V path.  I think we are missing a test for that.

Reply:
Oh, nevermind, the test was under `CodeGenHLSL/builtins`.